
export const createCoupon = async (req, res) => {
    try {
        return res.send('Sharma');
    } catch (error) {
        console.log('Coupon Error', error);
        
    }
}

export const getAllCoupon = async (req, res) => {
    try {
        return res.send('Sharma');
    } catch (error) {
        console.log('Coupon Error', error);
        
    }
}

export const getCouponById = async (req, res) => {
    try {
        return res.send('Sharma');
    } catch (error) {
        console.log('Coupon Error', error);
        
    }
}

export const editCouponById = async (req, res) => {
    try {
        return res.send('Sharma');
    } catch (error) {
        console.log('Coupon Error', error);
        
    }
}

export const deleteCouponById = async (req, res) => {
    try {
        return res.send('Sharma');
    } catch (error) {
        console.log('Coupon Error', error);
        
    }
}